build/deps/deps.jsonc

```diff
@@ -84,14 +86,15 @@
       "name": "com_google_protobuf",
       "owner": "protocolbuffers",
       "repo": "protobuf",
-      "file_regex": "protobuf-.*\\.tar\\.gz"
+      "file_regex": "protobuf-.*\\.tar\\.gz",
+      "freeze_version": "v29.3"
```

Comment:
I think freezing at this version is reasonable based on protobuf version management being difficult/us not being able to update further downstream atm (longer story), but let's add a comment